README.md

```diff
@@ -34,11 +34,11 @@ People get addicted to Netdata. Once you use it on your systems, there's no goin
 ![Users who are addicted to
 Netdata](https://user-images.githubusercontent.com/1153921/96495792-2e881380-11fd-11eb-85a3-53d3a84dcb29.png)
 
-> **[Latest release](https://github.com/netdata/netdata/releases/latest): v1.30.0, March 31, 2021**
+> **[Latest release](https://github.com/netdata/netdata/releases/latest): v1.31.0, May 19, 2021**
 >
-> The v1.30.0 release of Netdata brings major improvements to our packaging and completely replaces Google Analytics/GTM
-> for product telemetry. We're also releasing the first changes in an upcoming overhaul to both our dashboard UI/UX and
-> the suite of preconfigured alarms that comes with every installation.
+> The v1.31.0 release of Netdata comes with re-packaged and redesigned elements of the dashboard to help you focus on
```

Comment:
For some reason, I find this sentence super tiring. I would prefer bullets, but that's a style thing.